core/src/test/java/org/opensearch/sql/expression/operator/predicate/BinaryPredicateOperatorTest.java

```diff
@@ -615,4 +633,12 @@ public void compare_int_long() {
     FunctionExpression equal = DSL.equal(DSL.literal(1), DSL.literal(1L));
     assertTrue(equal.valueOf(valueEnv()).booleanValue());
   }
+
+  @ParameterizedTest
+  @MethodSource("getCidrArguments")
+  public void test_cidr(String address, String range, boolean expected) {
+    FunctionExpression cidr = DSL.cidr(DSL.literal(address), DSL.literal(range));
+    assertEquals(cidr.type(), BOOLEAN);
```

Comment:
Thanks. I've removed these tests now, since this is also (and more appropriately) tested in `IPFunctionTest`. I believe that test class has all its `assert*` arguments in the correct order.